src/Resources/public/js/autocomplete-create.js

```diff
@@ -0,0 +1,119 @@
+function createAutoCompleteCreateFields() {
+    var autocompleteCreateFields = $('[data-easyadmin-autocomplete-add-url]');
+
+    autocompleteCreateFields.each(function () {
+        var $this = $(this),
+            url = $this.data('easyadmin-autocomplete-add-url'),
+            url_action = $this.data('easyadmin-autocomplete-action-url')
+            field_name = $this.data('easyadmin-autocomplete-field-name');
+            button_text = $this.data('easyadmin-autocomplete-button-text');
+            
+        var select_id = $this.attr('id');
+
+        $this.select2({
+            theme: 'bootstrap',
+            ajax: {
+                url: url,
+                dataType: 'json',
+                delay: 250,
+                data: function (params) {
+                    return { 'query': params.term, 'page': params.page };
+                },
+                // to indicate that infinite scrolling can be used
+                processResults: function (data, params) {
+                    return {
+                        results: data.results,
+                        pagination: {
+                            more: data.has_next_page
+                        }
+                    };
+                },
+                cache: true
+            },
+            placeholder: '',
+            allowClear: true,
+            minimumInputLength: 1,
+            language: {
+                noResults: function () {
+                    return '<a href="#" class="btn btn-info" onclick="ajaxModalEntityAction(\''+url_action+'\', \''+select_id+'\', \''+field_name+'\');return false;">'+button_text+' '+field_name+'</a>';
+                }
+            },
+            escapeMarkup: function (markup) {
+                return markup;
+            }
+        });
+    });
+}
+function ajaxModalEntityAction(url_action, select_id, field_name) {
+    $('#'+select_id).select2('close');
+    $.ajax({
+        url : url_action,
+        type: 'GET',
+        success: function(data) {
+            showModalEntityForm(data, url_action, field_name, select_id);
+            $('#modal-entity-form').modal({ backdrop: true, keyboard: true });
+        }
+    });
+}
+function showModalEntityForm(data, url_action, field_name, select_id) {
+    var content = $('#modal-entity-form p.modal-body-content');
```

Comment:
I think `content` should be `modalBody` and `p.modal-body-content` may be removed :

```javascript
var modalBody = $('#modal-entity-form .modal-body');
modalBody.html(data.html);
```